src/lib/perf_histograms/rpc.ml

```diff
@@ -42,4 +42,38 @@ module Plain = struct
       implement_multi ?log_not_previously_seen_version
         (f |> deocorate_impl ~name)
   end
+
+  (* Since currently we are using the same version of data type for all the
+     nodes in the testnet, so the sizes of decoding and encoding should be
+     the same. If in the future, we have different versions, we need to make
+     the metrics record sizes of decoding and encoding separately. *)
+  module Decorate_bin_io (Rpc : Intf.Versioned_rpc.S) = struct
+    let bin_read_response buf ~pos_ref =
```

Comment:
You could extend this with `include Rpc`, and all the passed-in functions would also be exported, except for the shadowed ones. And then you don't have to qualify everything with `Rpc.` below.